src/test_typing_extensions.py

```diff
@@ -3531,7 +3534,9 @@ def test_basic_plain(self):
 
     def test_repr(self):
         Ts = TypeVarTuple('Ts')
-        if TYPING_3_11_0:
+        if TYPING_3_12_0:
+            self.assertEqual(repr(Unpack[Ts]), 'typing.Unpack[Ts]')
+        elif TYPING_3_11_0:
```

Comment:
Hmm, I feel like the test failure is pointing to a real issue here. We should backport https://github.com/python/cpython/pull/104048 -- the current `typing_extensions.Unpack` repr doesn't make any sense if you use `Unpack` with a `TypedDict` as per PEP 692

Reply:
See #163